clients/venice-admin-tool/src/main/java/com/linkedin/venice/Arg.java

```diff
@@ -176,7 +176,10 @@ public enum Arg {
   FABRIC_B("fabric-b", "fb", true, "The name of the second fabric in store comparison."),
   SOURCE_FABRIC("source-fabric", "sf", true, "The fabric where metadata/data copy over starts from"),
   DEST_FABRIC("dest-fabric", "df", true, "The fabric where metadata/data gets copy over into"),
-  DEFAULT_CONFIGS("default-configs", "dc", false, "Use default store configs (intended for system stores)"),
+  FABRIC_GROUP(
```

Comment:
Is --fabric_group the right option to use, or should we use --fabric option with --force?

Which prod fabric will be chosen as source fabric when fabric_group is specified to prod? If that fabric is the one that has data loss, we will end up using a bad fabric to repush stores.

Reply:
It makes sense. I will add the support for --fabric and --force so that a specific source fabric can be specified for recovery command.